src/components/bc-start.ts

```diff
@@ -1,15 +1,62 @@
-import {customElement} from 'lit/decorators.js';
-import {BitcoinConnectElement} from './BitcoinConnectElement';
-import {withTwind} from './twind/withTwind';
-import {html} from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
```

Comment:
minor: could you install prettier in vscode so we can keep the same formatting? I just pushed an update in the master branch so that it'll pick prettier by default for vscode